local-dev/templates/firecloud-orchestration.conf.ctmpl

```diff
@@ -64,7 +64,7 @@ akka {
 auth {
 	authorityEndpoint = "https://terra{{ $b2cTenant }}b2c.b2clogin.com/terra{{ $b2cTenant }}b2c.onmicrosoft.com/v2.0?p=b2c_1a_signup_signin_{{ $b2cEnv }}"
 	oidcClientId = "{{ $b2cAppId }}"
-	legacyGoogleClientId = "{{ $refreshTokenCredential.web.client_id }}"
+	b2cProfileWithGoogleBillingScope = "b2c_1a_signup_signin_billing_{{ $b2cEnv }}"
```

Comment:
Nice, I guess prior to this Rawls and Orch swagger don't have a way to request GCP billing scopes via B2C..

Reply:
Does this need some helmfile update as well?